src/apb/engine.py

```diff
@@ -1232,6 +1232,9 @@ def cmdrun_test(**kwargs):
         kwargs['tag']
     )
 
+    registry = get_registry(kwargs)
```

Comment:
If you have a look at the `cmdrun_run` you'll see we set the tag based on the registry like:

```
    apb_project = kwargs['base_path']
    registry = get_registry(kwargs)
    spec = get_spec(apb_project)
    tag = registry + "/" + kwargs['namespace'] + "/" + spec['name']

    image = build_apb(
        apb_project,
        kwargs['dockerfile'],
        tag
    )
    push_apb(registry, tag)
```

I think you will need to do something like that in order for this to work correctly.